### PR TITLE
Update docs and change name of python exporter

### PR DESCRIPTION
**Pull request by plutasnyy**

### CODE REVIEW

File: tutorials/sst/README.md [hunk 1]
```diff
@@ -17,32 +21,92 @@ or:
...
 ```python
 """
-This is the first cell, in it you can use all the things that are available in typical markdown:
-**bold**, *italic*, code, url, images, headers, lists etc.
+This is the first cell,here you are able to use whole syntax that is available in typical markdown. For example
+**bold**, *italic*, `code`, url, images, headers, lists etc.
+"""
+```
+
+Everything except triple quotes is treated as code cell, however you can add multiple markdown cells after each other:
```
Comment (Adamage) [inline tutorials/sst/README.md:39]: very nice.

File: tutorials/sst/README.md [hunk 2]
```diff
@@ -17,32 +21,92 @@ or:
...
+
+## Transformation to other formats
+By default, we expect in the repository store beyond python file that is single source of truth :
+- jupyter notebook which was not executed 
+- markdown file with the outputs after the execution
+- pure python file without documentation
+
+You can automatically generate all 3 formats by using the command:
+```bash
+sst convert2all --source path_to_you_python_file
```
Comment (Adamage) [inline tutorials/sst/README.md:83]: output ?

File: tutorials/sst/README.md [hunk 3]
```diff
@@ -17,32 +21,92 @@ or:
...
+ --execute
+ ```
+
+You can also define the output type by specifying the appropriate extension in output:
+```bash
+sst convert \
+ --source myfile.py \
+ --output outputfile.md \
+ --no-execute
+ ```
```
Comment (Adamage) [inline tutorials/sst/README.md:110]: Please add

```
You can convert a whole batch of python scripts, using a yaml configuration file:
```bash
sst batch_convert \
 --config tutorial_list.yml \
--input-dir tutorials/tutorials \
 --output-dir tutorials/tutorials/my_batch \
 --execute
 ```
Config file would look like this:
```
tutorials:
    - name: Tutorial1
      source: pytorch/tutorial1/script.py
```

**Review by Adamage — CHANGES_REQUESTED** [review]

File: tutorials/sst/README.md (hunk 2 above)
Comment (plutasnyy) [reply tutorials/sst/README.md:83]: is not neccessary

Comment (plutasnyy) [reply tutorials/sst/README.md:83]: but I will add

**Review by Adamage — APPROVED** [review]

**Pull request merged**